gc-design-system-variable-proposal.css

```diff
@@ -0,0 +1,167 @@
+}
+/* Example: */
+.grid-with-gutter {
+    margin: calc(var(--gd-grid-gutter-value, var(--gc-grid-gutter-value)) * -1);
+}
+.grid-column-1of3 {
+    padding: var(--gd-grid-gutter-value, var(gc-grid-gutter-value));
+}
+
+/* Typography Variables ===================================================== */
+/* These variables are used to define standard typographic elements. Note the the color of typography is covered in the color section below. */
+:root {
+    /* Font Types - to help avoid using multiple fonts inconsistently, we can set default fonts for serif, sans-serif, monospace, and script. This doesn't prevent other fonts from being used, but it allows us to set the fonts on components and maintain that consistency across the experience. */
+    --gc-font-sans: "Arial", sans-serif;
+    --gc-font-sans-serif: var(--gd-font-sans, var(--gc-font-sans)); /* Sometimes people want to type out the whole name, so this is repeated for their sake. */
+    --gc-font-serif: "Times New Roman", serif;
+    --gc-font-mono: "Consolas", monospace;
+    --gc-font-monospace: var(--gd-font-mono, var(--gc-font-mono)); /* Same idea here. */
+    --gc-font-script: script;
+    /* We can define a universal line height. */
+    --gc-font-line-height: 1.5;
+    /* This variable should be used on the HTML. It tells the site to respect the user's browser font size setting at 100%. Typically this variable can be set to something slightly lower than 100% on smaller devices, and something slightly higher on larger devices. Providing a variable for this allows the site to cater to different audiences (maybe they want their default font size to always be 150% of the user's setting to ensure visibility). */
+    --gc-font-size-setting: 100%;
+    /* Now we set the default font size. This variable allows us to manipulate the font scale using a multiplier, similarly to the margins and padding above. */
+    --gc-font-size-default: 1rem;
+    /* Now the multiplier (this creates a proper typography scale, Google "type scales" for more info. */
+    --gc-font-size-scale: 1.333;
+    --gc-font-size-caption: calc(var(--gd-font-size-default, var(--gc-font-size-default)) / var(--gd-font-size-scale, var(--gc-font-size-scale)));
+    --gc-font-size-copy: var(--gd-font-size-default, var(--gc-font-size-default));
+    --gc-font-size-h5: calc(var(--gd-font-size-default, var(--gc-font-size-default)) * var(--gd-font-size-scale, var(--gc-font-size-scale)));
+    --gc-font-size-h4: calc(var(--gd-font-size-h5, var(--gc-font-size-h5)) * var(--gd-font-size-scale, var(--gc-font-size-scale)));
+    --gc-font-size-h3: calc(var(--gd-font-size-h4, var(--gc-font-size-h4)) * var(--gd-font-size-scale, var(--gc-font-size-scale)));
+    --gc-font-size-h2: calc(var(--gd-font-size-h3, var(--gc-font-size-h3)) * var(--gd-font-size-scale, var(--gc-font-size-scale)));
+    --gc-font-size-h1: calc(var(--gd-font-size-h2, var(--gc-font-size-h2)) * var(--gd-font-size-scale, var(--gc-font-size-scale)));
```

Comment:
Note: In GCWeb the font height is fixed for all heading

Reply:
This could be changed. An issue has been opened https://github.com/canada-ca/design-system/issues/25 and should be promoted in the wet-boew/GCWeb repo instead